mage_ai/frontend/components/Backfills/Table/index.tsx

```diff
@@ -88,7 +89,17 @@ function BackfillsTable({
         total_run_count: totalRunCount,
       }, idx) => {
         const arr = [
-          <Text default key="status" monospace>{status || 'inactive'}</Text>,
+          <Text
+            danger={BackfillStatusEnum.FAILED === status}
+            default={BackfillStatusEnum.CANCELLED === status}
+            info={BackfillStatusEnum.INITIAL === status}
```

Comment:
`info` prop is status is `initial` or `running`. This is the blue color.

Reply:
Honestly speaking, I copied the styles from the `BlockRunsTable` (because you told me they are similar, but it's still my bad), so should they be different? E.g. the `running` status' colour is yellow on `BlockRunsTable`, but blue here.

Reply:
Yeah, we can go with the color scheme I mentioned in my other comment. We will probably make an update in a separate PR to make the colors consistent across the different tables (e.g. block runs and triggers as well).